provider/cmd/pulumi-resource-docker-build/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "docker-build",
   "displayName": "docker-build",
-  "version": "0.0.2-alpha.1714063884+13fb5b61",
+  "version": "1.0.0-alpha.0+dev",
```

Comment:
Shouldn't this be `0.0.2-alpha.x`?

Reply:
This can be customised in the makefile by setting the default PROVIDER_VERSION. If you're currently working towards 1.0 then this would seem to make sense, but I can modify the makefile to be something like 0.1.0.1-alpha+dev if you think it's clearer.